components/automate-ui/src/app/ui.component.ts

```diff
@@ -58,7 +60,9 @@ export class UIComponent implements OnInit {
         : true;
     });
 
+    // Initial calls for polled events
     this.store.dispatch(new GetIamVersion());
+    this.store.dispatch(new GetSomeUserPerms({ paths: [ProjectConstants.APPLY_RULES_ENDPOINT] }));
```

Comment:
... And this does the initial call at application start (required because `observableInterval` emits at the _end_ of the interval).
Notice the other initial call already there to get the version.